drivers/i2c/slave/Kconfig

```diff
@@ -0,0 +1,65 @@
+# Kconfig - I2C Slave configuration options
+
+#
+# Copyright (c) 2017 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# I2C options
+#
+menuconfig I2C_SLAVE
+	bool
+	prompt "I2C Slave Drivers"
+	depends on I2C
+	default n
+	help
+	  Enable I2C Slave Driver Configuration
+
+if I2C_SLAVE
+
+config I2C_SLAVE_INIT_PRIORITY
+	int
+	default 60
+	prompt "Init priority"
+	help
+	  I2C Slave device driver initialization priority.
+
+config SYS_LOG_I2C_SLAVE_LEVEL
+	int
+	prompt "I2C Slave log level"
+	depends on SYS_LOG
+	default 0
+	help
+	  Sets log level for I2C drivers.
+	  Levels are:
+	  - 0 OFF, do not write
+	  - 1 ERROR, only write SYS_LOG_ERR
+	  - 2 WARNING, write SYS_LOG_WRN in addition to previous level
+	  - 3 INFO, write SYS_LOG_INF in addition to previous levels
+	  - 4 DEBUG, write SYS_LOG_DBG in addition to previous levels
+
+config I2C_SLAVE_EEPROM
```

Comment:
I think you can move i²c slave eeprom stuff into its own Kconfig.eeprom. 
If your api gets in - and it will - there will be a lot more i²c slaves I hope here.

Reply:
OK